### PR TITLE
Update DNS

### PR DESCRIPTION
**Pull request by kimg45**
- link to quad9 profile download page
- remove GUI app for hosts file that hasn't been updated in over a year
- remove blocklists that haven't been updated in years
- remove GUI apps for dnscrypt that haven't been updated in years

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,16 +402,14 @@ You may also wish to enable [additional security options](https://github.com/drd
 
 macOS 11 introduced "DNS configuration profiles" to configure encrypted DNS, filter domains and use DNSSEC.
 
-DNS profiles [can be created](https://dns.notjakob.com/) or obtained from providers such as [Quad9](https://docs.quad9.net/), [AdGuard](https://adguard-dns.io/en/public-dns.html) and [NextDNS](https://nextdns.io/).
+DNS profiles [can be created](https://dns.notjakob.com/) or obtained from providers such as [Quad9](https://docs.quad9.net/Setup_Guides/MacOS/Big_Sur_and_later_%2528Encrypted%2529/#download-profile), [AdGuard](https://adguard-dns.io/en/public-dns.html) and [NextDNS](https://nextdns.io/).
```
Comment (drduh) [inline README.md:405]: The encoded characters don't work for me. Can we use https://docs.quad9.net/Setup_Guides/MacOS/Big_Sur_and_later_(Encrypted)/#download-profile

Comment (kimg45) [reply README.md]: done

**Review by drduh — APPROVED** [review]

**Comment by beerisgood** [comment]
I‘m too late but DNS profiles aren’t compatible with Apple's PrivateRelay. Also it is the most secure solution for encrypted DNS.

**Comment by kimg45** [comment]
Does it make you turn off private relay when you install a profile?

**Comment by beerisgood** [comment]
> Custom DNS settings
> If a user has configured custom-encrypted DNS settings using a profile or an app, the DNS server specified will be used instead of ODoH. Safari connections and all unencrypted HTTP connections will also resolve names using the specified DNS server prior to routing through Private Relay.
>
> An unencrypted DNS server provided by a local network or manually edited in Settings (iOS) or System Preferences (macOS) will not be used for iCloud Private Relay traffic.

https://www.apple.com/privacy/docs/iCloud_Private_Relay_Overview_Dec2021.PDF


What did you mean with it doesn’t cover all traffic? PrivateRelay isn’t restricted to Safari but the whole system.

**Comment by kimg45** [comment]
>iCloud Private Relay is designed to protect your privacy by ensuring that when you browse the web in Safari, no single party — not even Apple — can see both who you are and what sites you're visiting.
https://support.apple.com/en-us/102602
I only see it mentioning Safari in the documentation.

**Comment by beerisgood** [comment]
Looks like Quad9 doesn’t tell the truth here. 
as you’re wrote, only Safari can use Private Relay. At least today.

**Comment by kimg45** [comment]
Thought so, glad we confirmed it.

**Pull request merged**